### DISCUSSION TITLE
How to document interviews in EMu

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#577] [HOW TO...]
When and how should I record interviews (either with just audio or video)?

[google doc](https://docs.google.com/document/d/13GAasn3d25h371KSi-J-354O3koC-RXKYucLe1Iu8lY)

**Comment by rondlg** [comment]
@magpiedin We should update the document (still says draft)!

**Reply by magpiedin** [reply]
fixed

**Comment by rondlg** [comment]
@magpiedin Updates to the interview tab in v9 may alter this workflow.

See: https://github.com/fieldmuseum/EMu-Documentation/issues/153

**Reply by magpiedin** [reply]
Noted [here in workflow](https://docs.google.com/document/d/13GAasn3d25h371KSi-J-354O3koC-RXKYucLe1Iu8lY/edit#heading=h.w3yxuqnvkwwp)
